src/visualizers/SeqColor.ts

```diff
@@ -0,0 +1,452 @@
+            this.keyboardEvents()
+        }
+    }
+    /** The following code can be used to fix the keyboardEvents() bug
+     * once issue #120 is resolved
+	keyPressed() {
+  if (this.sketch.keyCode === this.sketch.LEFT_ARROW) {
+        // Show/hide description box when the left arrow key is pressed
+            if (this.boxIsShow == true && this.subG != null) {
+                this.boxIsShow = false
+                this.undrawBox()
+            } else {
+                this.boxIsShow = true
+                this.drawBox()
+            }
+	}  else if (this.sketch.keyCode === this.sketch.RIGHT_ARROW) {
+        // Show/hide label when right arrow key is pressed
+    if (this.showLabel == false) {
+                this.showLabel = true
+                this.drawLabel()
+            } else {
+                this.showLabel = false
+                this.undrawLabel()
+            }
+	}
+	}
+	**/
+    keyboardEvents() {
+        // Show description box when the left arrow key is pressed
+        if (this.sketch.keyIsDown(this.sketch.LEFT_ARROW)) {
+            if (this.boxIsShow == true && this.subG != null) {
+                this.boxIsShow = false
+                this.undrawBox()
+            } else {
+                this.boxIsShow = true
+                this.drawBox()
+            }
+        }
+
+        // Show label when right arrow key is pressed
+        if (
+            this.sketch.keyIsDown(this.sketch.RIGHT_ARROW)
+            && this.firstDraw == false
+        ) {
+            if (this.showLabel == false) {
+                this.showLabel = true
+                this.drawLabel()
+            } else {
+                this.showLabel = false
+                this.undrawLabel()
+            }
+        }
+
+        // Increase brightness "[" key is pressed
+        if (this.sketch.keyIsDown(219) && this.firstDraw == false) {
+            this.brightnessUp()
+        }
+
+        // Decrease brightness "]" key is pressed
+        if (this.sketch.keyIsDown(221) && this.firstDraw == false) {
+            this.brightnessDown()
+        }
+    }
+
+    brightnessUp() {
+        this.brightAdjust += 1
+        this.redrawCircles()
+    }
+
+    brightnessDown() {
+        this.brightAdjust -= 1
+        this.redrawCircles()
+    }
+
+    // Draw labels for each circle
+    drawLabel() {
+        this.position = this.sketch.createVector(
+            this.initialPosition.x,
+            this.initialPosition.y
+        )
+        this.subL = this.sketch.createGraphics(
+            this.canvasSize.x,
+            this.canvasSize.y
+        )
+        this.subG.colorMode(this.sketch.HSB)
+        this.subG.noStroke()
+        for (let i = this.seq.first; i < this.n; i++) {
+            this.showCircleLabel(i)
+            this.changePosition()
+        }
+    }
+
+    // Remove all labels by drawing circles again
+    undrawLabel() {
+        this.redrawCircles()
+    }
+
+    redrawCircles() {
```

Comment:
When we toggle the labels and the legend-box, sometimes the redraw doesn't clear the background, so we end up with circles over top of the white label box.

Reply:
Again, moot because keyboard interaction has been removed. Resolving.